web/src/components/organisms/Settings/Members/hooks.ts

```diff
@@ -37,6 +39,23 @@ export default ({ workspaceId }: Props) => {
   const me = { id: data?.me?.id, myWorkspace: data?.me?.myWorkspace.id };
   const workspaces = data?.me?.workspaces as Workspace[];
 
+  const checkOwner = useCallback(() => {
+    const members = currentWorkspace?.members;
+    if (members) {
+      for (let i = 0; i < members.length; i++) {
+        if (members[i].userId === me?.id && members[i].role === "OWNER") {
+          return true;
+        }
+      }
```

Comment:
Please do something like this (please test this and fix if I have any errors in my suggestion):
```
const isOwner = useMemo(()=> currentWorkspace?.members.find(m => m.userId === me?.id && m.role === "OWNER", [currentWorkspace?.members, me?.id])
```

Reply:
Fixed, and tested.